README.md

```diff
@@ -40,16 +40,18 @@ The most basic usage involves just creating a Bunyan logger which writes raw obj
 const bunyanDebugStream = require('bunyan-debug-stream');
 
 const log = bunyan.createLogger({
-    name: "myLog",
-    streams: [{
-        level:  'info',
-        type:   'raw',
-        stream: bunyanDebugStream.create({
-            basepath: __dirname, // this should be the root folder of your project.
-            forceColor: true
-        })
-    }],
-    serializers: bunyanDebugStream.serializers
+  name: 'myLog',
```

Comment:
Just making sure – this reformatting was automatic:

![image](https://user-images.githubusercontent.com/1962469/187684267-3b209465-4802-47b0-92f2-93750c060af9.png)

Reply:
Apparently when I added prettier, I ran it on `src` but not on anything else.  :P